Atividade exclusiva

matheusbzevedo opened the pull request.

matheusbzevedo committed:
Quantidade a ser selecionada pelo usuário

matheusbzevedo committed:
1.1

matheusbzevedo committed:
1.2

funcionando a parte das parcelas e etc

anytitleyoulike commented on application/controllers/agendamento.php:
```diff
@@ -602,4 +602,15 @@ public function exclusiva($parametro){
 		}
 	}
+
+	public function teste(){
```
o nome do método é teste mesmo? altera isso

anytitleyoulike commented on application/views/agendamento/index.php:
```diff
@@ -641,10 +657,34 @@
+			var preco = $("input[name='preco_raw'").val() * $('#select-exclusiva option:selected').val();
+			$.ajax({
+				type: "POST",
+				url: "../agendamento/teste",
```
define um nome pra função que não seja teste

anytitleyoulike commented on application/views/agendamento/index.php:
```diff
@@ -641,10 +657,34 @@
+	<script>
 		function validaDesconto(){
 			var cupom_desconto = $("input[name='cupom_desconto']").val();
+			var quantidade = ($("input[name='evento_codigo']").val() == 1659) ? $('#select-exclusiva option:selected').val() : $("input[name='quantidade']").val();
```
isso aqui ta valendo só pra uma atividade??

matheusbzevedo committed:
correções

matheusbzevedo committed:
1.3

Redução da quantidade de 5 pra 2.
Inserção da opção "SELECIONE" nas parcelas para evitar "bug".
